api/controllers/restaurantController.js

```diff
@@ -57,13 +58,27 @@ const getRestaurants = async (req, res) => {
 };
 
 const getRestaurant = async (req, res) => {
-  const { restaurantId } = req.params;
-  const restaurant = await Restaurant.findByPk(restaurantId, {include: ['cuisines', 'reviews']});
-  if (restaurant) {
-    res.send(restaurant);
-  } else {
-    res.send(mockRestaurants[0]);
-  }
+  const { APIID } = req.params;
+  const endpoint = `https://api.yelp.com/v3/businesses/${APIID}`;
```

Comment:
Change APIID to yelpId

```suggestion
  const endpoint = `https://api.yelp.com/v3/businesses/${yelpId}`;